spec/unit/loga/log_subscribers/action_mailer_spec.rb

```diff
@@ -6,7 +6,7 @@
   subject(:mailer) { described_class.new }
 
   let(:event) do
-    double('event', payload: payload, duration: 0.0001, time: Time.now)
+    instance_double('event', payload: payload, duration: 0.0001, time: Time.now)
```

Comment:
This should be ` instance_double(Loga::Event, ..`

Reply:
This in fact a `ActiveSupport::Notifications::Event` but all the more reason to fix it. I will also configuring mocks more strictly.